Makefile

```diff
@@ -50,6 +50,9 @@ update: deps api api-docs app-sre-saas-template
 verify: update staticcheck fmt vet promtool
 	git diff-index --cached --quiet --ignore-submodules HEAD --
 	git diff-files --quiet --ignore-submodules
+	echo "WTK"
+	git diff --exit-code HEAD --
```

Comment:
I'm trying to figure out what auto-generated changes [the verify presubmit is asking for][1], but this code doesn't seem to be running.  At least, I don't see any `WTK` output before the command exits a few lines down...

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_hypershift/1954/pull-ci-openshift-hypershift-main-verify/1603153782781251584#1:build-log.txt%3A290

Reply:
And I was finally able to drag out a patch by base64-encoing it so the wrapping Make/ci-operator business wouldn't garble the output: 03dfcc25bba2d41151f8cea13fc57d0d5de5e71b .  But obviously it would be nice to have a more accessible approach...